src/simulator_core/entities/assets/demand_cluster.py

```diff
@@ -45,14 +45,15 @@ def __init__(
         self,
         asset_name: str,
         asset_id: str,
+        geometry: Any,
```

Comment:
As mentioned before I would not add the geometry call to all the different assets; if we make a development plot routine where we can just set these values it might work better?